<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>781</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
    background-color: #000000;

}
</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 30pt &quot;Segoe UI&quot;;
   color: #85E2FF;
    background-color: #000000;
}</string>
    </property>
    <property name="text">
     <string>Staff Workspace</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>91</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Brown Simple Cute Catering Logo (2).png);</string>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>761</width>
     <height>411</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>781</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
    background-color: #f7f7f7;

}
</string>
     </property>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>611</width>
        <height>241</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>199</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>30</number>
      </attribute>
      <column>
       <property name="text">
        <string>Name of the Staff</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>User</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Password</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>400</y>
        <width>111</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #000000;
    color: #ffffff;
    border-radius: 6px;
}

QPushButton:hover {
    background-color: #333333;
    color: #ffffff;
}
</string>
      </property>
      <property name="text">
       <string>+ Create Account</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>70</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #000000;
    color: #ffffff;
    border-radius: 6px;
}

QPushButton:hover {
    background-color: #333333;
    color: #ffffff;
}
</string>
      </property>
      <property name="text">
       <string>+ Create Account</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #85E2FF;
    color: #000000;
    border-radius: 6px;
}

QPushButton:hover {
    background-color: #33CCFF;
}
</string>
   </property>
   <property name="text">
    <string>Accounts</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #85E2FF;
    color: #000000;
    border-radius: 6px;
}

QPushButton:hover {
    background-color: #33CCFF;
}
</string>
   </property>
   <property name="text">
    <string>Student review</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #85E2FF;
    color: #000000;
    border-radius: 6px;
}

QPushButton:hover {
    background-color: #33CCFF;
}
</string>
   </property>
   <property name="text">
    <string>Dashboard</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="and.qrc"/>
 </resources>
 <connections/>
</ui>
